tests/util.py

```diff
@@ -80,27 +80,24 @@ def make_alternating_lattice(lattice_size, angle=0, extra_shell=2):
     return points_1, points_2
 
 
-def make_fcc(nx=1, ny=1, nz=1, scale=1.0, noise=0.0):
-    """Make an FCC crystal for testing
-
-    :param nx: Number of repeats in the x direction, default is 1
-    :param ny: Number of repeats in the y direction, default is 1
-    :param nz: Number of repeats in the z direction, default is 1
-    :param scale: Amount to scale the unit cell by (in distance units),
-        default is 1.0
-    :param noise: Apply Gaussian noise with this width to particle positions
-    :type nx: int
-    :type ny: int
-    :type nz: int
-    :type scale: float
-    :type noise: float
-    :return: freud Box, particle positions, shape=(nx*ny*nz, 3)
-    :rtype: (:class:`freud.box.Box`, :class:`np.ndarray`)
+def make_cubic(nx=1, ny=1, nz=1, fractions=np.array([[0, 0, 0]],
+               dtype=np.float32), scale=1.0, noise=0.0):
+    """Make a cubic crystal for testing.
+
+    Args:
+        nx: Number of repeats in the x direction, default is 1.
+        ny: Number of repeats in the y direction, default is 1.
+        nz: Number of repeats in the z direction, default is 1.
+        fractions: The basis to replicate using the lattice.
+        scale: Amount to scale the unit cell by (in distance units)
+               (Default value = 1.0).
+        noise: Apply Gaussian noise with this width to particle positions
+               (Default value = 0.0).
+
+    Returns:
+        tuple (py:class:`freud.box.Box`, :class:`np.ndarray`): freud Box,
```

Comment:
Missing colon before `:py:class:`; inconsistent with use of `:py:class:` vs. `:class:` (assumed Python namespace).